R/gen_functions.R

```diff
@@ -183,8 +186,19 @@ gen_temperature <- function(world,
     hot_plus_r <- dist_to_hot[this_hot] + hotspot_radius[this_hot]
     hot_modifier <- hot_minus_r / (hot_plus_r * 2)
 
+    # Calculate the noise
+    if (is.null(noise)) {
+      heat_noise <- 0
+    } else {
+      heat_noise <- noise$amplitude * gen_simplex(
```

Comment:
I couldn't find a function definition for `gen_simplex()`. I think it belongs to the package ambient? If yes, the package name must be prepended, like so: `ambient::gen_simplex(...)`.